frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalsystem/project/MacheteProjectResolver.java

```diff
@@ -55,15 +52,17 @@ public boolean cancelTask(ExternalSystemTaskId taskId, ExternalSystemTaskNotific
     return false;
   }
 
-  @SuppressWarnings({"guieffect:call.invalid.ui", "interning:not.interned"})
+  @SuppressWarnings("interning:not.interned")
   private Boolean hasSelectedGitMacheteTab(Project project) {
     var toolWindowManager = ToolWindowManager.getInstance(project);
     var toolWindow = toolWindowManager.getToolWindow(ToolWindowId.VCS);
     if (toolWindow != null) {
-      var contentManager = toolWindow.getContentManager();
-      var gitMacheteContent = contentManager.findContent("Git Machete");
-      return contentManager.getSelectedContent() == gitMacheteContent;
+      var contentManager = toolWindow.getContentManagerIfCreated();
```

Comment:
Btw you can mark these methods as `@Nullable` in stubs for safety in future (Checker isn't very sound if it comes to assumptions about methods coming from compiler jars, see https://github.com/typetools/checker-framework/issues/4079)